package.json

```diff
@@ -18,49 +18,65 @@
     "emscripten",
     "geolocation"
   ],
-  "main": "index.js",
+  "main": "dist/h3-js.js",
+  "umd:main": "dist/h3-js.umd.js",
+  "browser": "dist/h3-js.umd.js",
```

Comment:
This one will be replaced by `h3-js.browser.js` in a subsequent diff.

Reply:
Definitely looking forward to that one. UMD is an abomination.

Reply:
Generally agreed, but it's useful for standalone use in `script` tags, etc.

Reply:
Out of curiosity how will the "browser" version be different to the UMD version?

Reply:
I'm putting this diff up shortly - we're actually going to use `browser` to remap the CommonJS and ES6 bundles to builds that use the Emscripten browser environment build. This removes the references to `fs` and `path` that you noted in #21, and should simplify Webpack builds. So `browser` will actually be used for browser builds in external rollup tools, rather than for direct use in the browser.

Reply:
I shall keep an eye out for it and give a quick test with Rollup JS + Webpack when its up...